feature_annotations.yaml

```diff
@@ -0,0 +1,24 @@
+# This code-annotations configuration file supports OEP-17, Feature Toggles.
+
+source_path: ./
+report_path: reports
+safelist_path: .annotation_safe_list.yml
+coverage_target: 50.0
+annotations:
+  feature_toggle:
+    - ".. feature_toggle_name:":
+    - ".. feature_toggle_type:":
+        choices: [switch, rollout, group, ConfigurationModel, feature_flag]
+    - ".. feature_toggle_default:":
+    - ".. feature_toggle_description:":
+    - ".. feature_toggle_category:":
+    - ".. feature_toggle_use_cases:":
+        choices: [incremental_release, launch_date, monitored_rollout, graceful_degradation, beta_testing, vip, opt_out, open_edx]
```

Comment:
Uses cases could be more than one of these.  I don't think the tool supports that yet.

Reply:
@robrap It does!  You can separate your values with commas or spaces.